resources/ext/cli/ca.erb

```diff
@@ -0,0 +1,30 @@
+#!/opt/puppetlabs/puppet/bin/ruby
+
+require 'fileutils'
+require 'puppetserver/ca/stub'
+
+command = ARGV[0]
+
+unless command == "import"
+  STDERR.puts "  'import' is the only valid subcommand, try:"
+  STDERR.puts "    `puppetserver ca import <key> <bundle> <crl>"
```

Comment:
^ these are supposed to be provided as paths to the files? (for key, bundle, crl)